ClinGen/clincoded
# Connects to #1526. Adding PAGE study population data.

**Steps to test:**
1. Login with Auth0 authentication and select the **13685** and/or **130281** ClinVar IDs in the VCI.
2. Upon being taken to the **evidence-only** view of the VCI, click on the **Population** tab.
3. Scroll down the page and expect to see the **PAGE** data table with data correlating to the 11 super populations.
4. Click on the **PAGE Study** anchor link and expect to be shown the **PAGE** credit at the bottom of the tab.
5. Click on the **GGV Browser** link and expect to be taken to the external GGV Browser site.
6. Now try adding a different ID -- **CA10824140** CA ID -- in the VCI.
7. Proceed to the **Population** tab and expect to see _NO_ data being returned in the PAGE data table but a "No population data was found for this allele in PAGE. Search GGV for this variant." message being shown.
8. Logout and click on the **Demo Login** button to sign in as a demo user.
9. Repeat Steps 1 & 2.
10. Scroll down the page and expect to see the **PAGE** data table with only a message indicating the PAGE study population data not being available to demo users.
